unison-share-api/src/Unison/Server/Types.hs

```diff
@@ -239,21 +248,44 @@ data NamedType = NamedType
 instance ToJSON NamedType where
   toEncoding = genericToEncoding defaultOptions
 
+instance FromJSON NamedType where
+  parseJSON = genericParseJSON defaultOptions
+
 deriving instance ToSchema NamedType
 
 instance ToJSON TermTag where
   toJSON = \case
-    Doc -> "doc"
-    Test -> "test"
-    Plain -> "plain"
+    Doc -> "Doc"
```

Comment:
I believe I mistakenly changed the JSON serialization of these tags in a previous commit which hasn't been deployed yet 😬 

This should change them back.